### DISCUSSION TITLE
DDS Security with Newly generated self signed CA,permissions CA and Test User certificates not working (with Openssl CA)

### DISCUSSION THREAD

**Discussion by Thirumalkumar530** [OpenDDS/OpenDDS#3981] [Q&A]
Hi,
I'm Trying opendds security with new pair of certificates. I have generated the following with Openssl 
1. Self Signed Identity CA 
2. Self Signed Permissions CA
3. Test User certificates signed by Identity CA
I have taken Governance File and permissions File which was provided by OpenDDS project,Edited them (changed Domain ID Range in Governance file and changed Subject name in permission file) and signed with above permissions CA certificate. When i use the above files for testing the dds application with security it is failing to create domain participant. We have created two sample application both publisher and subscriber and tested them with default certificates and files(governance and permission) provided by opendds project and it is working fine with default files. 
 Is it required to change Governance file if so what changes are expected in order to make it work.
Kindly help in this regard.

**Comment by mitza-oci** [comment]
Please attach the log files, signed permissions, and CA cert.
